AUTHORS

```diff
@@ -10,6 +10,7 @@ Andy Goldstein <agoldste@redhat.com>
 Anton Tiurin <noxiouz@yandex.ru>
 Antonio Mercado <amercado@thinknode.com>
 Arnaud Porterie <arnaud.porterie@docker.com>
+Arthur Baars <arthur@semmle.com>
```

Comment:
Please don't include authors file updates in PRs. We will update this automatically at release time.